roles/cfg_openwrt/templates/corerouter/config/airos-dfs-reset.j2

```diff
@@ -1,8 +1,13 @@
 #jinja2: trim_blocks: "true", lstrip_blocks: "true"
+config airos-dfs-reset general
```

Comment:
Please ship an empty config in case of no devices.

Reply:
done